client/ayon_core/pipeline/publish/abstract_collect_render.py

```diff
@@ -214,8 +217,11 @@ def process(self, context):
             data = self.add_additional_data(data)
             render_instance_dict = attr.asdict(render_instance)
 
-            instance = context.create_instance(render_instance.name)
-            instance.data["label"] = render_instance.label
+            if render_instance.source_instance is not None:
+                instance = render_instance.source_instance
```

Comment:
Hmm, maybe we should remove the source instance from the data.
```suggestion
                instance = render_instance.source_instance
                render_instance_dict.pop("source_instance")
```

Reply:
Or maybe just this
```python
instance = remder_instance_dict.pop("source_instance")
if instance is None:
    instance = context.create_instance(render_instance.name)
```

Reply:
Agreed. Implemented with https://github.com/ynput/ayon-core/pull/405/commits/c538ed58efd1c35849455822c089dc53b2ca8731